### PR TITLE
Update wait on ntp

### PR DESCRIPTION
**Pull request by johnsonw**
ntp should be stopped before invoking sntp and waiting for
synchronization.

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2063)

### CODE REVIEW

File: vagrant/scripts/wait_for_ntp.sh [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 NTP_SERVER=$1
 
+systemctl stop ntpd.service
+
+sntp -s "$NTP_SERVER"
 sntp -s "$NTP_SERVER"
 
 systemctl restart ntpd.service
```
Comment (jgrund) [inline vagrant/scripts/wait_for_ntp.sh:10]:
```suggestion
systemctl start ntpd.service
```

File: vagrant/scripts/enable_debug.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+touch /tmp/chroma-agent-debug
```
Comment (jgrund) [inline vagrant/scripts/enable_debug.sh:2]: This should have a provisioner in the Vagrantfile as well

Comment (johnsonw) [reply vagrant/scripts/enable_debug.sh:2]: I moved the debug work into a separate patch and added provisioners to the vargrant file.

**Comment by johnsonw** [comment]
The update ntp logic was merged into docker-snapshots. I moved the debugging work to a separate patch so i'm closing this pr.

**Pull request closed without merging**